registry/storage/driver/rados/rados.go

```diff
@@ -471,6 +471,7 @@ func (d *driver) Delete(ctx context.Context, objectPath string) error {
 		totalSize, err := d.getXattrTotalSize(ctx, oid)
 
 		if err != nil {
+			err = d.deleteOid(objectPath)
```

Comment:
@stevvooe , i intend to remove the object path when the rados doesn't write the content completely. When rados doesn't write the content completely, do we need to remove that content firstly? or just remain the uncompleted content? i think we should remote the uncomplete content.